internal/confmapprovider/discovery/testdata/invalid-properties.d/properties.discovery.yaml

```diff
@@ -0,0 +1 @@
+invalid
```

Comment:
Nit: should there be an empty newline at the end of this file?

Reply:
added one for good measure.

Reply:
I don't notice either way except for GitHub reviews, since it explicitly shows if a file doesn't have an empty line at the end. Since most do I just assume that's the standard, so this comment was just a consistency check.